game/game/gamescript.cpp

```diff
@@ -1128,6 +1128,14 @@ void GameScript::invokePickLock(Npc& npc, int bSuccess, int bBrokenOpen) {
   vm.call_function<void>(fn, bSuccess, bBrokenOpen);
   }
 
+void GameScript::invokeRefreshAtInsert(Npc& npc) {
+  auto fn = vm.find_symbol_by_name("B_RefreshAtInsert");
+  if(fn==nullptr)
```

Comment:
it would be good idea to cache function in `GameScript`, similar to `ItKE_lockpick`, as there will be burst of this calls after "sleep".